acceptance_tests/features/environment.py

```diff
@@ -138,3 +137,9 @@ def get_survey_type(tags):
         return "Social"
 
     return 'Business'
+
+
+def setup_sequential_data_for_test():
```

Comment:
not sure this belongs here?

Reply:
I'm not sure where it belongs, we don't have any specific place for sequential test data setup so putting it with the hook where it's called seemed to make sense?

Reply:
doesn't make a lot of difference as it will eventually be deleted but thought maybe fixures.py more appropriate as all setup happens there?